src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -26,6 +26,7 @@
   <ItemGroup>
     <PackageReference Include="Apache.Arrow" Version="0.14.1" />
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
+    <PackageReference Include="Microsoft.Data.Analysis" Version="0.2.0" />
```

Comment:
Is the version number important here? If I stop specifying it, will we always look for the latest version or do we stop looking if an older version is found in a local Nuget cache?